lib/streamlit/hashing_py3.py

```diff
@@ -18,11 +18,48 @@
 
 import dis
 import importlib
+import inspect
+import textwrap
+
+from streamlit.errors import UserHashError
+
+
+def _hashing_user_error_message(exc, lines, filename, lineno):
+    return (
+        """
```

Comment:
Add comment above this string saying this needs to have zero indentation otherwise `%(line)s` will render incorrectly in Markdown.